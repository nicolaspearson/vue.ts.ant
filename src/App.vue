<template>
  <div id="app">
    <PageHeader />
    <router-view />
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

import PageHeader from '@/components/PageHeader.vue';

@Component({
  components: {
    PageHeader,
  },
})
export default class App extends Vue {}
</script>

<style lang="scss">
@import url('https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i');
@import './scss/typography';

#app {
  font-family: 'Lato', sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: $black;
}
#nav {
  padding: 30px;
  a {
    font-weight: bold;
    color: $black;
    &.router-link-exact-active {
      color: $primary;
    }
  }
}
</style>
