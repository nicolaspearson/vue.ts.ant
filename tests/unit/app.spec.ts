import { shallowMount } from '@vue/test-utils';

import App from '@/App.vue';

describe('App.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(App, {
      propsData: {},
      components: {
        'router-view': {
          name: 'router-view',
          render: (h: any) => h('div'),
        },
      },
      stubs: ['PageHeader'],
    });
    expect(wrapper).toMatchSnapshot();
  });
});
