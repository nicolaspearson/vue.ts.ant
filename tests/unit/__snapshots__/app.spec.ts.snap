// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue matches snapshot 1`] = `
<div id="app">
  <pageheader-stub></pageheader-stub>
  <router-view-stub></router-view-stub>
</div>
`;
